gateway-ha/pom.xml

```diff
@@ -84,8 +104,8 @@
         </dependency>
         <dependency>
             <groupId>com.sun.mail</groupId>
-            <artifactId>javax.mail</artifactId>
-            <version>1.6.2</version>
+            <artifactId>jakarta.mail</artifactId>
+            <version>${jakarta.mail.version}</version>
```

Comment:
All the dependencies and their versions should be declared in the top level pom .. and then used here without version